src/components/SortForm.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+
+const SortForm = ({ setSearchParams }) => {
+  const sortTerms = ["created_at", "comment_count", "votes"];
+  const order = ["asc", "desc"];
+
+  const [searchTerm, setSearchTerm] = useState({
+    sort_by: "created_at",
+    order: "desc",
+  });
+
+  const handleChange = (event) => {
+    setSearchTerm({ ...searchTerm, [event.target.name]: event.target.value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setSearchParams(searchTerm);
+  };
+
+  return (
+    <form onSubmit={handleSubmit} className="sortForm">
+      <label>
+        Sort By:
+        <select name="sort_by" id="sort_by" onChange={handleChange}>
```

Comment:
This select is currently uncontrolled. Please make sure that the select's value is tied to your state object

Reply:
No problem, I'll start working on them. Should I do another pull request on slack once I've done that?